galaxy.yml

```diff
@@ -8,34 +8,38 @@ namespace: community
 name: sap_launchpad
 
 # The version of the collection. Must be compatible with semantic versioning
-version: 1.1.0
+version: 1.2.0
 
 # The path to the Markdown (.md) readme file. This path is relative to the root of the collection
 readme: README.md
 
 # A list of the collection's content authors. Can be just the name or in the format 'Full Name <email> (url)
 authors:
-  - IBM Lab for SAP Solutions
-  - IBM Cloud for SAP
-  - IBM Consulting for SAP
-  - MatthiasWinzeler
+  - Sean Freeman <sean.freeman(at)uk.ibm.com>
+  - Sheng Li Zhu <digglife@gmail.com>
+  - Matthias Winzeler <https://github.com/MatthiasWinzeler>
+  - Marcel Mamula <marcel.mamula(at)suse.com>
 
 ### OPTIONAL but strongly recommended
 # A short summary description of the collection
-description: Collection of Ansible Modules for SAP Launchpad APIs
+description: Collection of Ansible Modules and Roles for downloading SAP Software
```

Comment:
Same as above, larger remit than download - and the potential to extend much further with SAP Notes etc